helper.psm1

```diff
@@ -11,6 +11,7 @@ If(-Not(Test-Path -PathType Container -Path "work"))
     New-Item -ItemType Directory -Path "work"
 }
 
+$global:WIRESHARKPATH = "c:\Program Files\Wireshark"
```

Comment:
Why not `(Get-Command tshark.exe).Path`?

Reply:
its not in the path - so this won't find it.